doc/guide.rst

```diff
@@ -201,6 +201,28 @@ What this does is tell pyOptSparse that the key ``obj_name`` in the function ret
 For optimizers that can do multi-objective optimization (e.g. NSGA2), multiple objectives can be added.
 Optimizers that can only handle one objective enforce that only a single objective is added to the optimization description.
 
+Analytic Derivatives
+++++++++++++++++++++
+
+pyOptSparse can automatically compute derivatives of the objective and constraint functions using finite differences or the complex-step method (See implementation details `here <https://mdolab-pyoptsparse.readthedocs-hosted.com/en/latest/api/gradient.html>`__).
```

Comment:
These can all be replaced by internal Sphinx links, e.g. :ref:`here <gradient>` (I think..)

Reply:
I am not quite familiar with such commands. Plus, it might be good to be consistent with the other sections. So let's leave them for later if we finally decide to make such changes.